javascript/javascript1/week4/char_frequencies.js

```diff
@@ -0,0 +1,26 @@
+ //Character frequency counter
+
+ 
+ function getCharacterFrequencies(word, char){
+    
+    let count=0;
+ 
+    characters=[{character:"", count:0}];
+     if(typeof(word)==="string"){
+       
+         for(let i=0; i<word.length; i++){
+              
+              if(word[i]===char){
+                
+                 count +=1;
+                  
+               }    
+         }   
+     } 
+    
+     return count; 
+ }
+ console.log(getCharacterFrequencies("ababacedae", 'a'));
```

Comment:
Your function should be called like this ` console.log(getCharacterFrequencies("ababacedae"));` And should return
`{
  characters: [
    {
      character: 'a',
      count: 4
    },
    {
      character: 'b',
      count: 2
    },
    {
      character: 'c',
      count: 1
    },
    {
      character: 'd',
      count: 1
    }
  ], length: 5
}`